runtime-modules/project-token/src/tests/transfer.rs

```diff
@@ -1025,6 +1025,33 @@ fn issuer_transfer_fails_with_non_existing_source() {
     })
 }
 
+#[test]
+fn issuer_transfer_fails_with_non_existing_dst_member() {
```

Comment:
Hmm, interesting, this looks like it's actually coming from https://github.com/Joystream/joystream/pull/3796, but some of the changes are still missing (for example, `transfer_fails_with_non_existing_dst_member` test case)

Reply:
I added back all the changes in that PR: f5d8b50a3ce09be67cdb721e88f790f3b6be8d3d